plugins/de.cognicrypt.core/src/de/cognicrypt/core/Constants.java

```diff
@@ -537,6 +538,7 @@ public String toString() {
 	public static final String SHOW_SECURE_OBJECTS = "de.cognicrypt.staticanalyzer.secureObjects";
 	public static final String CALL_GRAPH_SELECTION = "de.cognicrypt.staticanalyzer.callgraphSelection";
 	public static final String SELECT_CUSTOM_RULES = "de.cognicrypt.staticanalyzer.selectCustomRules";
+	public static final String SUPPRESS_TLS_ERRORS = "de.cognicrypt.staticanalyzer.suppressTLSErrors";

```

Comment:
This option should be more general than just for TLS. Rename not just here, but throughout the project, please.

Reply:
@kruegers so should it just be suppress_errors?

Reply:
@kruegers also the name of the checkbox should be changed? If I get it right, there should be a checkbox in prefpage which suppress all unwanted errors. right?

Reply:
That would be too general. The name should reflect what kinds of errors are suppressed.

Reply:
should it be categorized to the type of error, like constraint errors? or more into this category?